app/api/enrolments.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional
+
+from fastapi import APIRouter, Depends, HTTPException
+
+from app.domain.entities.enrolment_request import State
+from app.repositories.s3_course_repo import S3CourseRepo
+from app.repositories.s3_enrolment_repo import S3EnrolmentRepo
+from app.requests.enrolment_requests import ProcessEnrolmentRequest
+from app.security import oauth2_scheme
+from app.use_cases import get_enrolments as ge
+from app.use_cases import process_enrolments as pe
+
+router = APIRouter()
+course_repo = S3CourseRepo()
+enrolment_repo = S3EnrolmentRepo()
+
+
+@router.get("/enrolment_requests")
+def get_enrolment_requests(
```

Comment:
who will post this request? ACP employer?
But he has already posted an enrolment request in TP-enrolments.
https://github.com/ACWIC/training-provider-enrolment/blob/main/app/api/enrolments.py